content/html/concepts/attributes/terms/dir/dir.md

```diff
@@ -0,0 +1,64 @@
+---
+
+Title:
+
+-  'dir'
+
+Description:
+
+-  'Specifies the text direction of the content of an element.'
+
+Subjects:
+
+-  'Web Development'
+
+-  'Web Design'
+
+Tags:
+
+-  'Attribute'
+
+-  'Value'
+
+Catalog Content:
+
+-  'learn-html'
+
+-  'paths/front-end-engineer-career-path'
+
+---
+
+The **`dir`** global attribute specifies the text direction of an HTML element's content.
+
+The `dir` attribute is used to set the base direction of text for display. It is essential to support languages that use right-to-left scripts. Many different languages are written with these scripts, including Adlam, Arabic, Dhivehi, Hebrew, Mandinka, N'Ko, Pashto, Persian, Pular, Sindhi, Syriac, Thaana, Urdu, Yiddish, etc.  
+
+## Syntax:
+
+```html
+<element  dir="ltr | rtl | auto">
+```
+
+## Possible Attribute Values:
```

Comment:
```suggestion
The `dir` attribute has the following possible values:

```

Reply:
When I submitted the original pull request, you suggested I write it this way. I think it looks better too. But if you still prefer, Ill change it

Reply:
@RobinLilly Let's do it this way.